test/unit/atomic/test-atomic-ref-constructor.cpp

```diff
@@ -9,15 +9,16 @@
 /// Source file containing tests for atomic ref constructors (and use of getPointer for verification)
 ///
 
-#include <RAJA/RAJA.hpp>
-#include "RAJA_gtest.hpp"
+#include "RAJA/RAJA.hpp"
 
-#include "test-atomic-ref.hpp"
+#include "RAJA_gtest.hpp"
 
 #if defined(RAJA_ENABLE_CUDA)
-#include "RAJA_unit_forone.hpp"
+#include "RAJA_test-forone.hpp"
```

Comment:
Renaming this file is fine, but I wanted to point out that it's original intent was to provide a CUDA launching construct specifically for our unit tests, so that we can do CUDA unit tests without using our own forall's.

Reply:
@rchen20 Sorry. I didn't complete my renaming exercise to make things clearer. I'll change the name of this file to RAJA_unit-test-forone.hpp